qutebrowser/utils/jinja.py

```diff
@@ -127,3 +130,33 @@ def render(template, **kwargs):
 
 environment = Environment()
 js_environment = jinja2.Environment(loader=Loader('javascript'))
+
+
+@functools.lru_cache()
+def template_config_variables(template: str) -> typing.FrozenSet[str]:
+    """Return a frozenset of config variables that is used in template."""
```

Comment:
```suggestion
    """Return the config variables used in the template."""
```

(the type is already conveyed by the annotations anyways)